supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -1038,7 +1038,7 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 // RHEL: General; kept as is
 [[rpm-ostree]]
 ==== image:images/yes.png[yes] rpm-ostree (noun)
-*Description*: A hybrid image or system package that hosts operating system updates.
+*Description*: A hybrid image or system package that hosts operating system updates. Use lowercase format and backticks, for example, &#34;``rpm-ostree``&#34;.
```

Comment:
```suggestion
*Description*: A hybrid image or system package that hosts operating system updates. Use the lowercase format with backticks. For example, &#34;``rpm-ostree``&#34;.
```

Reply:
Is the quotation code neede? I see the preview renders fine but I guess further publishing on various browsers might or might not cause issues.

Reply:
I read the comments on the original issue, and @bergerhoffer mentions that we don't want to use ASCIIDoc examples in the style guide. But when we mention backticks, that's a formatting description for ASCIIDoc or Markdown. I think it would be better to describe what we want the final format to look like ("fixed-width font" or "monospaced font") rather than the specifics of how we generate that formatting. 

For example:

> A hybrid image or system package that hosts operating system updates. Use lowercase formatting with a fixed-width or monospaced font. Example: `rpm-ostree`